userspace/libsinsp/sinsp.cpp

```diff
@@ -1458,14 +1480,14 @@ threadinfo_map_t::ptr_t sinsp::get_thread_ref(int64_t tid, bool query_os_if_not_
 
 		m_n_proc_lookups++;
 
-		if(m_n_proc_lookups == m_max_n_proc_lookups)
+		if(m_n_proc_lookups == get_max_n_proc_loopkups())
 		{
 			g_logger.format(sinsp_logger::SEV_INFO, "Reached max process lookup number, duration=%" PRIu64 "ms",
 				m_n_proc_lookups_duration_ns / 1000000);
 		}
 
-		if(m_max_n_proc_lookups < 0 ||
-		   m_n_proc_lookups <= m_max_n_proc_lookups)
+		if(get_max_n_proc_loopkups() < 0 ||
```

Comment:
see above

Reply:
oh...I'm noticing that `n_max_proc_lookup` can be negative to disable lookup. This is not currently handled in this PR. I will slightly modify the code